distribution/src/main/scripts/fscrawler

```diff
@@ -38,6 +38,7 @@ JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true"
 
 # Ensure UTF-8 encoding by default (e.g. filenames)
 JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"
+JAVA_OPTS="$JAVA_OPTS -Dsun.jnu.encoding=UTF-8
```

Comment:
This should not be part of this PR as it's coming in another one.

Reply:
Yes I understood. The previous PR was just approved but still open. That's why. I have to create a PR from patch. Either way I will have a clean PR.

Reply:
No worries. If I'm able to merge your commit first, you will just have to rebase.